src/Controller/ModulesController.php

```diff
@@ -184,16 +184,41 @@ public function view($id): ?Response
         $revision = Hash::get($response, 'meta.schema.' . $this->objectType . '.revision', null);
         $schema = $this->Schema->getSchema($this->objectType, $revision);
 
+        // REMOVE THIS
+        $schema['categories'] = [
+            [
+                'name' => 'geografia e viaggi',
+                'label' => 'Geografia e Viaggi',
+            ],
+            [
+                'name' => 'letteratura',
+                'label' => 'Letteratura',
+            ],
+            [
+                'name' => 'sport e giochi',
+                'label' => 'Sport e giochi',
+            ],
+        ];
+
         $object = $response['data'];
 
+        // set checked state of categories
```

Comment:
Can `$category['checked']` set be performed in `Helper` or `Form` utility class? 
It seems very `view` specific... 

ModulesController::view() is already doing too much stuff...